extension/src/DocumentFunctions.ts

```diff
@@ -22,3 +22,25 @@ export async function searchTextFile(DocumentUri: vscode.Uri, StartPosition: num
 
     return { foundNode: (foundOffset >= 0), foundAtPosition: foundOffset };
 }
+
+export function getLineEnding(document: vscode.TextDocument) {
```

Comment:
It's something with the naming of these functions that bothering me. This is really nit-picky so disregard this if you think it's silly :)

```suggestion
export function documentLineEnding(document: vscode.TextDocument): string {
```